holmes/core/tools.py

```diff
@@ -399,6 +401,8 @@ def interpolate_command(self, command: str) -> str:
         return interpolated_command
 
     def check_prerequisites(self):
+        self._status = ToolsetStatusEnum.ENABLED
```

Comment:
Please check this carefully! I *think* it's correct, but I don't have a full understanding of how toolset.enabled interacts with toolset._status (I think enabled indicates the user intent and _status is the result of the checks so this is fine.) I also only tested with the local cli not the in-cluster path

Reply:
@aantn can you explain why you added it?